health-k8s/src/main/scala/com/devsisters/shardcake/K8sPodsHealth.scala

```
@@ -1,11 +1,28 @@
 package com.devsisters.shardcake
 
+import com.coralogix.zio.k8s.client.{
```

I think you can simply import `_` at this point

---

I can, and will if you prefer, but top-level wildcard imports have a fair few downsides in Scala including increasing compilation times and compatibility hazards, e.g. a k8s client version bump that introduced a naming conflict or implicit conversion that is silently accepted but does the wrong thing. It's why I have my auto-import set to never wildcard.